modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go

```diff
@@ -39,6 +39,7 @@ func init() {
 //         root <path>
 //         split <at>
 //         env <key> <value>
+//         resolverootsymlink
```

Comment:
Change to `resolve_root_symlink`

Reply:
What about `resolve_symlinks`? I don't think `root` needs to be mentioned, and the internal golang function has the plural on it.

Reply:
I went with `resolve_root_symlink`: it's the root path that is resolved, not any other symlink inside it, so IMO being detailed here is correct.